docs/_include/variable_hydro.rst

```diff
@@ -0,0 +1,113 @@
+30 different depths over the 3m range. User defined logic then selects 
+the BEM dataset most closely corresponding to the instantaneous 
+heave position of the device. That new BEM dataset is then used to 
+calculate the hydrodynamic forcing on the device until updated again.
+
+Implementation
+""""""""""""""
+Variable hydrodynamics is implemented by allowing users to initialize a body
+with a cell array of h5 files, as many as required. Each i-th h5 file is then pre-processed into
+``body.hydroData(i)`` and ``body.hydroForce.hf{i}`` in the normal manner.
+The entirety of body.hydroForce is loaded into Simulink using a custom bus. 
+The index ``body1_hydroForceIndex`` in Simulink, which indexes 
+``body.hydroForce.hf{i}`` and selects the hydrodynamic force coefficients used
+in the hydrodynamic force calculations at a given time.
+Users *must* create their own logic and index signal in Simulink. A ``GoTo`` 
+block named ``body1_hydroForceIndex``, ``body2_hydroForceIndex`` must take 
+the signal so that the body block can read the value and select the correct 
+hydrodynamic coefficients.
+
+Variable hydrodynamics is body dependent and does not need to be applied to 
+every body in a simulation.
+To turn on variable hydrodynamics, use the ``body.variableHydro.option`` flag in the 
+wecSimInputFile and input multiple H5 files during the body initialization::
+
+    body(1) = bodyClass({'H5FILE_1.h5','H5FILE_2.h5','H5FILE_3.h5','H5FILE_4.h5','H5FILE_5.h5');
+    body(1).variableHydro.option = 1;
+
+
+If ``body.variableHydro.option=0`` any extra H5 files in the body initialization 
+are ignored. If only one H5 file is submitted to a file, variable hydrodynamics 
+are turned off.
+
+The flag ``body.variableHydro.hydroForceIndexInitial`` allows the user to set the
+default index used to define the hydrodynamic coefficients. This initial dataset
+should represent the body at the start of a simulation and likely its equilibrium 
+position. The initial dataset will be used to define the mass, center of gravity, 
+and center of buoyancy for a simulation.
+
+This parameter is flexible because an initial index of zero is not always convenient
+and can complicate indexing logic. For example, consider a flap-based device with
+multiple hydrodynamic datasets at various pitch angles (-50:10:50). It is most convenient
+to treat these angles in numerical order in BEM simulations, indexing logic, and other 
+data processing. In this case the initial position is at a pitch angle of 0 so ``body.variableHydro.hydroForceIndexInitial=6;``.
+
+Compatibility
+"""""""""""""
+Variable hydrodynamics is not compatible with certain WEC-Sim features. It cannot be used concurrently with:
+
+* State-space radiation calculations
+* FIR Filter radiation calculations
+* Generalized body modes within the same body. Seperate bodies could each have generalized body modes and variable hydrodynamics however.
+* Non-hydrodynamic and drag bodies
+
```

Comment:
Please convert this to a note. That's how we typically denote if a feature is not compatible with another. For example,  see PCT note here: https://wec-sim.github.io/WEC-Sim/main/user/advanced_features.html#parallel-computing-toolbox-pct